lab5/src/main/webapp/WEB-INF/admin-edit-events.jsp

```diff
@@ -0,0 +1,58 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core"%>
+<html lang="en">
+
+<head>
+
+    <meta charset="UTF-8">
+
+    <title>Incremental game framework - game information</title>
+
+    <link rel="stylesheet" href="../app.css">
+
+</head>
+<body class ="width-constraint">
+
+    <header>
+
+        <h1>Incremental game framework</h1>
+	</header>
+		<div class="row-container">
+
+            <form class="fill card" method= "POST">
+    
+                <div class="form-group">
+    
+                    <label for="name">Event Name</label>
+    
+                    <input type="text" id="names" name ="name">
```

Comment:
Information should be prefilled with the edit information with `value` attribute.